cinema/serializers.py

```diff
@@ -1 +1,134 @@
-# write serializers here
+from rest_framework import serializers
+from cinema.models import (
+    Order,
+    Ticket,
+    CinemaHall,
+    Actor,
+    Genre,
+    Movie,
+    MovieSession
+)
+
+
+class CinemaHallSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = CinemaHall
+        fields = ("name", "rows", "seats_in_row")
+
+
+class CinemaHallDetailSerializer(serializers.ModelSerializer):
+    def get_capacity(self, obj):
+        return obj.capacity
+
+    class Meta:
+        model = CinemaHall
+        fields = ("name", "rows", "seats_in_row", "capacity")
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    full_name = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Actor
+        fields = ("id", "first_name", "last_name", "full_name")
+
+    def get_full_name(self, obj):
```

Comment:
Type annotation is missing for the method `get_full_name`. According to the checklist, do not forget about type annotation in class methods.